.gitignore

```diff
@@ -16,14 +16,10 @@
 /packages/debian/pool/
 /packages/debian/dists/
 /packages/npm/
-scripts/node_modules/
+/scripts/node_modules
+.DS_Store
 
-# undocumented crap which is sitting on the production server. Is any of it
-# needed? it needs documenting and/or cleaning up.
+# artefacts from the /blog part of the site
 /blog/
-
-# should not be committed
-site.css
-docs/
-
-.DS_Store
\ No newline at end of file
+/site.css
+/docs
```

Comment:
is it true that this is from /blog?

Reply:
Not strictly for site.css. Probably doesn't need a comment at all?

Reply:
indeed. I'd prefer no comment over a misleading one.